### ISSUE TITLE
[MINVOKER-68] Enable accessibility for non-public members from BeanShell scripts

### ISSUE THREAD

**Issue by jira-importer** [apache/maven-invoker-plugin#355]
**[Benjamin Bentmann](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=bentmann)** opened **[MINVOKER-68](https://issues.apache.org/jira/browse/MINVOKER-68?redirect=false)** and commented

Accessing non-public fields/methods from BeanShell scripts currently requires one to invoke `setAccessibility( true )` at the start of the script. In contrast, this works out of the box for Groovy scripts. Furthermore, I doubt that anybody cares about accessibility during testing. So, for the sake of consistency with Groovy and the ease of testing, BeanShell's accessibility mode should be enabled, too.

---

**Affects:** 1.3

**Comment by jira-importer** [comment]
**[Benjamin Bentmann](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=bentmann)** commented

Done in [r700430](http://svn.apache.org/viewvc?view=rev&revision=700430).
